### ISSUE TITLE
Allow different system name for info_source

### ISSUE THREAD

**Issue by Enverex** [mickelson/attract#269]
Currently, the platform used to look-up content for the "info_source" is whatever you set as the "system" variable. Unfortunately this doesn't always work because, for example, if you have both Windows and DOS games set up, you may have PC/Windows set up separately to DOS, but this doesn't work with AttractMode because "DOS" isn't a valid system for thegamesdb.net, for this to work you'd need both lists set with the system to "PC" which obviously doesn't work.

In short, the platform for "info_source" needs to be separate to the "system" variable.

**Comment by mickelson** [comment]
The system variable can contain multiple systems (semicolon separated).  You should be able to input both windows and dos by having "Windows;DOS" (or whatever it is you want). Does it work for you if you do that?

**Comment by Enverex** [comment]
Actually this may be my mistake, I completely forgot the actual visible system name is in attract.cfg and isn't what's listed as "System" in the emulator config file. I'll rejig things and close this issue if I was wrong.
